qcelemental/util/internal.py

```diff
@@ -1,4 +1,4 @@
-from .._version import get_versions
+from qcelemental.extras import get_information
```

Comment:
how about `from ..extras import get_information`? it just seems untidy to import whole module from within.